clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10712,6 +10712,17 @@ static void addArgs(ArgStringList &DstArgs, const llvm::opt::ArgList &Alloc,
   }
 }
 
+static bool supportDynamicLinking(const llvm::opt::ArgList &TCArgs) {
+  if (TCArgs.hasFlag(options::OPT_fsycl_allow_split_with_dependencies,
```

Comment:
I would just inline this into the callsites and move this into a seperate function when the SYCL RT is updated, checking-in commented out code is not ideal IMO

Reply:
Removed commented code.  Changed into TODO comment.